docs/api/browser-window.md

```diff
@@ -1356,6 +1356,14 @@ removed in future Electron releases.
 **Note:** The BrowserView API is currently experimental and may change or be
 removed in future Electron releases.
 
+#### `win.setForwardMouseMessages(forward)` _Windows_
```

Comment:
We can probably make this an option of the `setIgnoreMouseEvents`, I don't think there are other usages of this API.

```
#### `win.setIgnoreMouseEvents(ignore[, options])`

* `ignore` Boolean
* `options` Object  (optional)
  * `forwardMouseMessages` Boolean (optional) - ...
```

And I think it is worthwhile to add a simple example for it, otherwise it is still hard to know how to make use of it.